app/src/main/java/org/p2p/wallet/solend/ui/info/SolendInfoSliderFragmentArgs.kt

```diff
@@ -0,0 +1,34 @@
+package org.p2p.wallet.solend.ui.info
+
+import android.os.Bundle
+import androidx.annotation.DrawableRes
+import androidx.annotation.StringRes
+import androidx.core.os.bundleOf
+import org.p2p.wallet.utils.requireInt
+
+private const val ICON_ARG = "sliderIcon"
+private const val TEXT_ARG = "sliderText"
+
+data class SolendInfoSliderFragmentArgs(
+    @DrawableRes val iconRes: Int,
+    @StringRes val slideText: Int,
+) {
+    fun toBundle(): Bundle = bundleOf(
+        ICON_ARG to iconRes,
+        TEXT_ARG to slideText
+    )
+
+    companion object {
+        @Throws(IllegalArgumentException::class)
+        fun fromBundle(bundle: Bundle?): SolendInfoSliderFragmentArgs {
+            requireNotNull(bundle) { "Required arguments is missing!" }
+
+            bundle.classLoader = SolendInfoSliderFragmentArgs::class.java.classLoader
```

Comment:
why do we need this anyways?

Reply:
not anyways but now android start to use it =)
![image](https://user-images.githubusercontent.com/99185280/195107546-50a5b9d6-e2aa-45eb-a379-7fd54471ada5.png)
so Classloader is important in some cases